pycbc/workflow/configuration.py

```diff
@@ -78,9 +124,13 @@ def resolve_url(url, directory=None, permissions=None, copy_to_cwd=True):
         elif copy_to_cwd:
             if os.path.isfile(filename):
                 # check to see if src and dest are the same file
-                src_inode = os.stat(u.path)[stat.ST_INO]
-                dst_inode = os.stat(filename)[stat.ST_INO]
-                if src_inode != dst_inode:
+                same_file = hash_compare(
+                    u.path,
+                    filename,
+                    chunk_size=hash_chunk_size,
+                    max_chunks=hash_max_chunks
+                )
+                if not same_file:
```

Comment:
Okay, so this solves the issue that sometimes `src` and `dest` are identified as the same file when they are not? (I'm not sure there's any risk if they are *not* identified as the same file, when they are).

I wonder why we need this check at all, it may be easier to just `shutil.copy` in any case?

Reply:
The copy can be one of the slowest parts of the workflow generation, so this solves an issue that they are not identified as the same when they are.

Like you said we can just do the copy without the check, but that removes this speed-up (this is mainly useful for testing)

It may be worth setting the `copy_to_cwd` kwarg default to `False`, or maybe allowing a `symlink_to_cwd` instead?

Reply:
Okay, let's merge this as is for now then. I'm not sure if I want to change the copying by default at this stage.